lib/src/winrt_helpers.dart

```diff
@@ -27,9 +27,21 @@ import 'winrt/foundation/winrt_enum.dart';
 /// Initializes the Windows Runtime on the current thread with a single-threaded
 /// concurrency model.
 ///
+/// This is usually not needed, as this package ensures that threads are
+/// implicitly assigned to the multi-threaded apartment (MTA). However, if you
+/// need your thread to be initialized with a single-threaded apartment (STA),
+/// you can call this function.
+///
 /// {@category winrt}
 void winrtInitialize() => RoInitialize(RO_INIT_TYPE.RO_INIT_SINGLETHREADED);
 
+/// Closes the Windows Runtime on the current thread.
+///
+/// {@category winrt}
+@Deprecated('Using this function may cause segfaults or runtime exceptions and '
```

Comment:
I think the text is a little strong here (and quite scary!)

If I understood it correctly, the source of our intermittent flake is that in a tight loop of tests, `RoUninitialize` might still be executing when the next test is running from a different isolate (but shared thread). And as such the next call to initialize COM may hit a race condition. The new approach simply initializes COM on error, and relies on the process termination to tear it down. 

My proposal is that we deprecate both `winrtInitialize` and `winrtUninitialize`, with the following comments, respectively:

```dart
@Deprecated('winrtInitialize is no longer required. The Windows Runtime is automatically initialized by the Dart projection if it is not already initialized on a given thread when a WinRT class is activated. If you explicitly want to initialize the current thread with a specific threading model, use RoInitialize directly instead. This function will be removed in the next major release.')
void winrtInitialize() {}

@Deprecated('winrtUninitialize is not required in most scenarios, since Windows will clean up the process on exit. If you explicitly want to uninitialize the Windows Runtime, use RoUninitialize directly instead. This function will be removed in the next major release.')
void winrtUninitialize() {}
```

I'd advocate that winrtUninitialize() points to an empty function, since there's no harm in it being removed.

Reply:
> If I understood it correctly, the source of our intermittent flake is that in a tight loop of tests, `RoUninitialize` might still be executing when the next test is running from a different isolate (but shared thread). And as such the next call to initialize COM may hit a race condition. The new approach simply initializes COM on error, and relies on the process termination to tear it down.

Yes, I believe that's what happened.

Thanks, these are great!